uptane-standard.md

```diff
@@ -874,22 +880,23 @@ If {{TAP-5}} is supported, a primary ECU SHALL download metadata and images foll
-    1. Locate and download a Targets metadata file from the Image repository that contains an image with exactly the same file name listed in the Director metadata, following the procedure in {{resolve_delegations}}.
-    2. Check that the Targets metadata from the Image repository matches the Targets metadata from the Director repository:
-        1. Check that the non-custom metadata (i.e., length and hashes) of the unencrypted image are the same in both sets of metadata.
-        2. Check that all "MUST match" custom metadata (e.g., hardware identifier and release counter) are the same in both sets of metadata.
-        3. Check that the release counter in the previous targets metadata file is less than or equal to the release counter in this targets metadata file.
+1. Load the repository mapping metadata ({{repo_mapping_meta}}).
+2. Check each mapping, in the listed order, and identify the first mapping that matches the requested image.
+3. Once a mapping is identified for the requested file, metadata is downloaded and verified from the assigned repositories listed in the mapping. To do so, an ECU SHALL perform the following steps:
+    1. Load the latest attested time from the time server, if implemented.
+    2. Download and check the Root metadata file from the repository, following the procedure in {{check_root}}.
+    3. Download and check the Timestamp metadata file from the repository, following the procedure in {{check_timestamp}}.
+    4. Download and check the Snapshot metadata file from the repository, following the procedure in {{check_snapshot}}.
+    5. Download and check the top-level Targets metadata file from the repository, following the procedure in {{check_targets}}.
+    6. If necessary, locate and download a Targets metadata file from the repository that contains an image with with the given file name, following the procedure in {{resolve_delegations}}. (This SHOULD NOT be necessary to perform on a Director repository, as its top-level Targets metadata SHOULD NOT have delegations.) Also, check that the release counter for the image in the custom targets metadata in the previous targets metadata file is less than or equal to the release counter in this targets metadata file.
+4. If the targets metadata is a match across the specified threshold of repositories, return this metadata. A match means that:
+    1. The non-custom metadata (i.e., length and hashes) of the unencrypted image are the same in all sets of metadata.
+    2. The "MUST match" custom metadata (e.g., hardware identifier and release counter) are the same in all sets of metadata.
```

Comment:
In the multi-director case, there's a new wrinkle: the ECU identifier MUST be the same across all Director repositories.

Reply:
(And ECU identifier cannot be in must-match metadata, because it doesn't exist on image repositories.)